### PR TITLE
Fix/9974 entry

### PR DESCRIPTION
**Pull request by RH-tj**
Fix CrashloopBackoff error for new FedRamp specific debug-container

context: APPSRE-9974

### CODE REVIEW

**Comment by RH-tj** [comment]
lgtm

File: debug-container-fedramp/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-=============================
-|| FedRamp Debug Container ||
-=============================
+========================
+ FedRamp Debug Container 
+========================
```
Comment (cubismod) [inline debug-container-fedramp/README.md:3]:
```suggestion
# FedRamp Debug Container 
```
GitHub markdown won't render the `======`

**Comment by cubismod** [comment]
> context: APPSRE-9974

File: debug-container-fedramp/Dockerfile [hunk 2]
```diff
@@ -46,9 +46,14 @@ RUN git clone --depth 1 https://github.com/brendangregg/perf-tools
...
+
 ENV TERM=vt100
 
 ENV PS1="\[\e[31m\]\[\e[0m\]\ndebug-fedramp\[\e[0;32m\]\u\[\e[m\]@\[\e[0;33m\]\h\[\e[m\]:\[\e[0;34m\]\w\[\e[m\] \$ "
 
+RUN chmod 777 /run.sh 
+
 USER 0
 
+ENTRYPOINT ["/run.sh"]
```
Comment (cubismod) [inline debug-container-fedramp/Dockerfile:59]: Can you add newlines to the end of your files including `run.sh`? It's not a big issue here but some of our tooling will complain and fail MRs if there is no newline. If you're using VSCode you can have the editor automatically do it with the preference: `files.insertFinalNewline` in your user settings

Comment (RH-tj) [reply debug-container-fedramp/Dockerfile:59]: great catch, thanks!  This is exactly the kind of thing that drives me nuts during CI testing

Comment (RH-tj) [reply debug-container-fedramp/Dockerfile]: Potential automation opportunity for me here: update the linter to do this automatically and/or block the MR if this is missing etc

**Review by cubismod — APPROVED** [review]
LGTM

**Pull request merged**